machine_learning/gaussian_naive_bayes.py

```diff
@@ -25,20 +27,26 @@ def main():
 
     # Gaussian Naive Bayes
     nb_model = GaussianNB()
-    nb_model.fit(x_train, y_train)
+    time.sleep(2.9)
```

Comment:
Why are we waiting for 3 seconds?!?  Comments needed to be added to explain such things.